test-apps/display-performance-test-app/src/frontend/TestRunner.ts

```diff
@@ -1021,58 +1022,43 @@ function getRenderOpts(curRenderOpts: RenderSystem.Options): string {
               case "EXT_frag_depth":
                 optString += "-fragDepth";
                 break;
-              default:
-                optString += `-${ext}`;
```

Comment:
@pmconne, you've got a few places in here (this is one) where it's getting rid of all of the defaults, so any new options that are made in core won't automatically get reflected in the test configurations as they have been--you'd have to make an intentional change to dpta itself to add in a new flag for the csv data or png file names. I'm guessing that's intentional, but I thought I should double check.

Reply:
Thanks @hnn0003 - that is intentional. We don't want the filenames to change unexpectedly - if we add a new option that we think should be included in the filename, we should add it explicitly. There are lots of options that have no effect on the image produced.